lib/gui/modules/drive-scanner.js

```diff
@@ -110,9 +95,16 @@ const scan = () => {
       includeSystemDrives: settings.get('unsafeMode')
     },
     usbboot: {
-      files: {
-        'bootcode.bin': USBBOOT_BOOTCODE_BIN_BUFFER,
-        'start.elf': USBBOOT_START_ELF_BUFFER
+      readFile: (name) => {
+        const blobPath = _.attempt(() => {
+          if (name === 'bootcode.bin' || name === 'start_cd.elf' || 'fixup_cd.dat') {
```

Comment:
I guess this was supposed to be `|| name === 'fixup_cd.dat') {` ? ;-)

Just in case we decide to use the non-cutdown start.elf in future (CD = Cut Down) I guess it probably doesn't hurt to add ` || name === 'start.elf' || name === 'fixup.dat'` in here too? (or perhaps we should only do that if / when we add start.elf)

Reply:
lol, great catch :P Yeah, lets add the conditional when we introduce the file.

Reply:
I guess we could even change this to:
```
if (_.includes(['bootcode.bin', 'start_cd.elf', 'fixup_cd.dat'], name)) {
```
:question:

Reply:
Good call. Done!